src/tools/manifest2proto.ts

```diff
@@ -0,0 +1,89 @@
+/**
+ * @license
+ * Copyright (c) 2020 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+import minimist from 'minimist';
+import fs from 'fs';
+import path from 'path';
+import {Runtime} from '../runtime/runtime.js';
+import {Manifest} from '../runtime/manifest.js';
+
+const opts = minimist(process.argv.slice(2), {
+  string: ['outdir', 'outfile'],
+  alias: {d: 'outdir', f: 'outfile'},
+  default: {outdir: '.'}
+});
+
+if (opts.help || opts._.length === 0) {
+  console.log(`
+Usage
+  $ tools/sigh manifest2proto [options] [file ...]
```

Comment:
Given that this script doesn't actually use protos, it might be better to call it `manifest2json`. It'll be an easy rename to do later if we make it actually use protos.